tests/import_test.go

```diff
@@ -0,0 +1,30 @@
+package tests_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/containerized-data-importer/tests/framework"
+)
+
+const (
+	testSuiteName   = "Importer Test Suite"
+	namespacePrefix = "importer"
+)
+
+var _ = Describe(testSuiteName, func() {
+	f := framework.NewFramework(namespacePrefix)
+
+	Describe("Creating standard PVC should not trigger importer", func() {
+		It("PVC should remain empty after being created", func() {
+			pvc, err := f.CreatePVCFromDefinition(f.Namespace.GetName(), framework.NewPVCDefinition("no-import", "1G", nil, nil))
```

Comment:
why use `GetName` vs `Namespace.Name`?

Reply:
Its a Java habit, where its always preferable to use the getter/setter over direct accessing the field, even if you have access to the field. This allows sub classes (in java) to override the setter/getter and get the correct behavior. Obviously it makes no difference in GO. @jeffvance is there a preference in GO to use the field over the getter?

Reply:
I honestly don't know if there's a go preference between direct access to an exported field vs. a getter method. I did a quick google search and came up with this [SO answer](https://stackoverflow.com/questions/45189126/go-getter-methods-vs-fields-proper-naming):
_Don't use getters and setters if your fields are exported. This just confuses the interface._